spec/features/user/user_can_connect_to_github_with_oauth_spec.rb

```diff
@@ -13,7 +13,7 @@
 
     visit '/dashboard'
 
-    VCR.use_cassette("dashboard") do
+    VCR.use_cassette("oauth") do
```

Comment:
I found a way we can set up cassettes using metadata, will talk about this cause I think it could save us some time. Everything looks great !